### PR TITLE
Addition of torch instance method torch.argwhere

### LINKED ISSUE #7859: argwhere

### PR DESCRIPTION
**Pull request by p-mishra1**
Close #7859

### CODE REVIEW

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hi @p-mishra1 
Thanks for contributing to Ivy😊

- Everything looks good except that the function `argwhere` is not implemented on the torch frontend so we can't call it from there. And as you can see that it is still in [the open task for contribution](https://github.com/unifyai/ivy/issues/9).

We can merge this PR once this function is implemented on the torch frontend.
Feel free to take on another open task or work on implementing this function 🙂

Best!

File: ivy/functional/frontends/torch/tensor.py [hunk 1]
```diff
@@ -331,6 +331,9 @@ def size(self, dim=None):
 
     def matmul(self, other):
         return torch_frontend.matmul(self._ivy_array, other)
+    
+    def argwhere(self):
+        return torch_frontend.argwhere(self._ivy_array)
```
[the remark this answers is not in the document]
Comment (p-mishra1) [reply ivy/functional/frontends/torch/tensor.py:336]: Thanks @hirwa-nshuti for pointing it out i was confused why the test case was failing, 
we have to add argwhere in [this](https://github.com/unifyai/ivy/blob/master/ivy/functional/frontends/torch/comparison_ops.py) frontend function right ?

[the remark this answers is not in the document]
Comment (hirwa-nshuti) [reply ivy/functional/frontends/torch/tensor.py:336]: No it has to be in this [file](https://github.com/unifyai/ivy/blob/master/ivy/functional/frontends/torch/indexing_slicing_joining_mutating_ops.py)

**Comment by hirwa-nshuti** [comment]
Hey good job so far, but we still need to write unit tests for the `argwhere` function.
Then I'll get the PR merged

**Comment by p-mishra1** [comment]
> Hey good job so far, but we still need to write unit tests for the `argwhere` function. Then I'll get the PR merged

@hirwa-nshuti can you give it a look

**Review by hirwa-nshuti — APPROVED** [review]
LGTM!
Thanks for contribution🙂

**Pull request merged**